tests/bluetooth/tester/overlay-le-audio.conf

```diff
@@ -149,3 +149,10 @@ CONFIG_BT_TBS_SUPPORTED_FEATURES=3
 
 # TMAP
 CONFIG_BT_TMAP=y
+
+#BAS
+CONFIG_BT_BAS=y
+CONFIG_BT_BAS_BLS=y
+CONFIG_BT_BAS_BLS_IDENTIFIER_PRESENT=y
+CONFIG_BT_BAS_BLS_BATTERY_LEVEL_PRESENT=y
+CONFIG_BT_BAS_BLS_ADDITIONAL_STATUS_PRESENT=y
```

Comment:
Doesn't this belong in the `prj.conf` file instead of the LE Audio overlay? BAS is not LE Audio related